rdr_service/dao/questionnaire_response_dao.py

```diff
@@ -692,7 +704,14 @@ def _update_participant_summary(
                                 and extension.get("valueCode") not in LANGUAGE_OF_CONSENT
                             ):
                                 logging.warning(f"consent language {extension.get('valueCode')} not recognized.")
-                    if getattr(participant_summary, summary_field) != new_status:
+                    elif self._is_digital_health_share_code(code.value):
+                        digital_health_sharing_status, something_changed = self._update_digital_health_status_field(
```

Comment:
Potentially introducing bugs: Did you intend to name this `something_changed` to take advantage of the logic that does some post-processing on the participant summary (down on line 740), or was that a coincidence? If none of the code for the block on 740 needs to run for this, can it be named something other than `something_changed`?

Reply:
Yes, I intend to do that for line 756 `session.merge(participant_summary)`.